src/main/java/pl/lukakan/contactform/MessageRepository.java

```diff
@@ -0,0 +1,28 @@
+package pl.lukakan.contactform;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+import org.thymeleaf.TemplateEngine;
+
+@Repository
+public class MessageRepository {
+    private final TemplateEngine templateEngine;
+    @Value("${spring.mail.username}")
+    private String domainMail;
```

Comment:
To też może być finalne, jęsli przyjmiesz do konstruktora dorzucisz parametr z adnotacją `@Value` i przypiszesz go do tego pola. Adnotacja w tym miejscu będzie oczywiście już niepotrzebna